services/base.js

```diff
@@ -250,29 +280,43 @@ class BaseJsonService extends BaseService {
       stripUnknown: true,
     })
     if (error) {
+      this.logDebug(sym.shrug, 'Response did not match schema', error.message)
       throw new InvalidResponse({
         prettyMessage: 'invalid json response',
         underlyingError: error,
       })
     } else {
+      this.logDebug(sym.bathtub, 'JSON after validation', value)
       return value
     }
   }
 
   async _requestJson({ schema, url, options = {}, notFoundMessage }) {
+    const logDebug = (...args) => this.constructor.logDebug(...args)
     if (!schema || !schema.isJoi) {
       throw Error('A Joi schema is required')
     }
-    return this._sendAndCacheRequest(url, {
+    const mergedOptions = {
       ...{ headers: { Accept: 'application/json' } },
       ...options,
-    })
+    }
+    logDebug(sym.bowAndArrow, 'request URL', url)
+    logDebug(sym.bowAndArrow, 'request options', mergedOptions)
```

Comment:
All the other log event titles start with a capital letter. Can we do the same with these ones too.. just for consistency